codegen/facelift/templates/ServiceIPC.template.h

```diff
@@ -68,7 +72,8 @@ class {{interface}}IPCAdapter: public facelift::IPCServiceAdapter<{{interface.fu
         facelift::IPCServiceAdapter<{{interface.fullyQualifiedCppType}}>::setService(srvc);
 
         {% for property in interface.properties %}
-        {% if property.type.is_model %}
+
+            {% if property.type.is_model %}
```

Comment:
Nitpick: actually I formatted it like that on purpose to make the generated code easier to read, small changes here can make it look more unstructured.